plugin/fzf.vim

```diff
@@ -304,7 +304,11 @@ try
     elseif type == 3
       let temps.input = tempname()
       call writefile(source, temps.input)
-      let prefix = 'cat '.s:shellesc(temps.input).'|'
+      if s:is_win
+          let prefix = 'type '.s:shellesc(temps.input).'|'
+      else
+          let prefix = 'cat '.s:shellesc(temps.input).'|'
+    endif
```

Comment:
Thanks. Can you change the block to a ternary expression? i.e. `(s:is_win ? 'type ' : 'cat '). ...`